aea/cli/register.py

```diff
@@ -55,4 +55,6 @@ def do_register(
     username = validate_author_name(username)
     token = register_new_account(username, email, password, password_confirmation)
     update_cli_config({AUTH_TOKEN_KEY: token})
+    if click.confirm('Do you want to subscribe?'):
```

Comment:
There need's to be a flag provided so you can override the prompt via the flag; the flag should be propagated to `init` as well